infrastructure/smart-contracts/contracts/upgrades/referee/Referee9.sol

```diff
@@ -475,8 +521,11 @@ contract Referee9 is Initializable, AccessControlEnumerableUpgradeable {
             amountClaimedByClaimers: 0
         });
 
-        // emit the events
-        emit ChallengeSubmitted(challengeCounter);   
+        // Only emit the challenge event if the challenge is not a batch challenge
+        if(_assertionId - _predecessorAssertionId == 1){
+            // emit the event
+            emit ChallengeSubmitted(challengeCounter);
+        } 
```

Comment:
I think we will want to emit this event no matter how many assertionIds it was for.

Reply:
Done!